[dev] Extension dependency grpc-tools v1.12.x does not work on Windows

Extension Name: vscode-gradle
Extension Version: v3.12.7
OS Version: Windows 11 22H2 (build 22621.2361)
VSCode version: v1.83.0

**Describe the bug**
As described in https://github.com/grpc/grpc-node/issues/2338, it seems that any version of `grpc-tools` greater or equal to `v1.12.0` is broken on Windows. However, it is possible to fix the problem by installing VS 2022 C++ build tools and Windows Universal CRT SDK. I think there could be two options to solve this issue:
1. Revert the `grpc-tools` version to the latest v1.11.x
2. Add the build tools requirements for Windows to the CONTRIBUTING.md documentation

I'll be glad to help out with a PR for this, so please let me know which one of the options above would be the best fit for the project 🙂 

**To Reproduce**
1. Use Windows machine
2. Run `./gradlew build` to build the project
4. Build fails on the grcp protoc execution

**Expected behavior**
Build should also work on Windows

**Screenshots**
N/A

**Output from "Gradle for Java"**
N/A

**Does the bug still exist if you disable all other extensions?**
N/A

**Additional context**
Build error output:
```shell
> Task :extension:npmInstall
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'vscode-gradle@3.13.0',
npm WARN EBADENGINE   required: { vscode: '^1.76.0', node: '^14.15.4', npm: '^6.14.10' },
npm WARN EBADENGINE   current: { node: 'v18.18.0', npm: '9.8.1' }
npm WARN EBADENGINE }

added 505 packages, and audited 506 packages in 11s

87 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

> Task :extension:generateProto FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':extension:generateProto'.
> protoc: stdout: . stderr: C:\Repos\vscode-gradle\extension\node_modules\grpc-tools\bin\protoc_plugin.js:37
      throw error;
      ^

  Error: Command failed: C:\Repos\vscode-gradle\extension\node_modules\grpc-tools\bin\grpc_node_plugin.exe

      at ChildProcess.exithandler (node:child_process:422:12)
      at ChildProcess.emit (node:events:517:28)
      at maybeClose (node:internal/child_process:1098:16)
      at ChildProcess._handle.onexit (node:internal/child_process:303:5) {
    code: 3221225781,
    killed: false,
    signal: null,
    cmd: 'C:\\Repos\\vscode-gradle\\extension\\node_modules\\grpc-tools\\bin\\grpc_node_plugin.exe'
  }

  Node.js v18.18.0
  --grpc_out: protoc-gen-grpc: Plugin failed with status code 1.


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.4/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 32s
14 actionable tasks: 14 executed
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -15,7 +15,7 @@ Start by opening an issue using one of the issue templates, or propose a change 
     - Add `npm_arch=x64` to $HOME/.gradle/gradle.properties
     - Add `protoc_platform=osx-x86_64` to $HOME/.gradle/gradle.properties
 6. If using Windows:
-    - The extension uses `grpc-tools@1.12.x` dependency which does not work out-of-the-box in Windows (check [this issues](https://github.com/grpc/grpc-node/issues/2338) for details), so you'll need to install some aditional DLLs.
+    - The extension uses `grpc-tools@1.12.x` dependency which does not work out-of-the-box in Windows (check [this issues](https://github.com/grpc/grpc-node/issues/2338) for details), so you'll need to install some aditional DLLs if the project build is failed.
     - Download and start [Build Tools for Visual Studio 2022](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022).
     - Go to the **Individual Components** tab and select the following:
       - `MSVC v143 - VS 2022 C++ x64/x86 build tools (latest)` (replacing `x64/x86` with your arch)

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -12,9 +12,16 @@ Start by opening an issue using one of the issue templates, or propose a change 
 3. Change directory to the root of the project
 4. Select Node version: `nvm use`
 5. If using an Apple M1:
-   - Add `npm_arch=x64` to $HOME/.gradle/gradle.properties
-   - Add `protoc_platform=osx-x86_64` to $HOME/.gradle/gradle.properties
-6. Build project files: `./gradlew build`
+    - Add `npm_arch=x64` to $HOME/.gradle/gradle.properties
+    - Add `protoc_platform=osx-x86_64` to $HOME/.gradle/gradle.properties
+6. If using Windows:
+    - The extension uses `grpc-tools@1.12.x` dependency which does not work out-of-the-box in Windows (check [this issues](https://github.com/grpc/grpc-node/issues/2338) for details), so you'll need to install some aditional DLLs.
+    - Download and start [Build Tools for Visual Studio 2022](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022).
+    - Go to the **Individual Components** tab and select the following:
+      - `MSVC v143 - VS 2022 C++ x64/x86 build tools (latest)` (replacing `x64/x86` with your arch)
+      - `Windows Universal CRT SDK`
+    - Click `Install` to add the components.
+7. Build project files: `./gradlew build`
 
 Running the build for the first time can take a bit of time, but subsequent builds should be fast.
 

